Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-slim
+
+ARG NAME
+ARG SPACY_MODEL
+WORKDIR /usr/bin/${NAME}
+
+COPY ./${NAME} /usr/bin/${NAME}
+
+RUN pip install pipenv
+RUN pipenv sync
+RUN if [ "$SPACY_MODEL" != "" ] ; then pipenv run python -m spacy download "$SPACY_MODEL" ; fi
```

Comment:
following our offline discussion - I'm in favor of spliting this into seperate dockerfiles (analyzer and anonymizer) where each has it our arguments. In this case, there's no reason for the anonymizer to have the space model version. And I assume further service-specific arguments (for future services we will convert) will be required. 
Let's continue this offline

Reply:
Completely legitimate option, had the same doubt.
What we can also do, is have a common dockerfile (without this line), and a default spacy in analyzer pip. Additionaly, there will be another dockerfile for building analyzer with a different spacy model, overriding the one defined in pip (it will essentially be the dockerfile suggested here). Downside is having to update both on changes.
I don't think that we will have much difference in the future since most of the differences can be handled with pip and environment variables.

Reply:
@SharonHart what do you mean by "overriding the one defined in pip"? Could you add some examples on how this would look like?

Reply:
Sure,
I think that we should use 'en_core_web_sm' as default, since it's faster for CI and working with locally. 
It will be part of analyzer pipenv, incorporated in the Pipfile and Pipfile.lock (making the dockerfile completely common).
Additionally, we can add another spacy model, installed with pipenv as an addition to en_core_web_sm, or override en_core_web_sm. We can control what the application uses with an environment variable.
It can be done in the separate dockerfile (if we decide to separate the services), or we can have additional one for reference only, for whomever wants to use a different one.

Reply:
vote for as many dockerfiles as required. keep the CI logic to minimum.